pipes/WDL/tasks/tasks_reports.wdl

```diff
@@ -447,7 +447,7 @@ task aggregate_metagenomics_reports {
 
 task MultiQC {
   input {
-    Array[File]    input_files = []
+    Array[File]    input_files
```

Comment:
I think the original thinking for this default empty assignment was so execution engines like Terra and dnanexus wouldn’t bother the user by default to fill it in if they wanted to omit it. That said… I can’t think off the top of my head where we’d currently have the multiqc task called in a workflow with inbound inputs so maybe this was pointless